### PR TITLE
GKE 1.6.0 benchmark

### PR DESCRIPTION
**Pull request by deboshree-b**
[CIS Google Kubernetes Engine (GKE) Benchmark v1.6.0 PDF.pdf](https://github.com/user-attachments/files/16755594/CIS.Google.Kubernetes.Engine.GKE.Benchmark.v1.6.0.PDF.pdf)

### CODE REVIEW

**Comment by rootxrishabh** [comment]
I am able to generate clusterpolicyreports with this PR on a GKE cluster. Will check test correctness next.

```
rootxrishabh@Rishabhs-MacBook-Air cfg % k get all                                        
NAME                                         READY   STATUS      RESTARTS   AGE
pod/kube-bench-adapter-startup-c12ab-ctkdv   0/1     Completed   0          6m28s

NAME                         TYPE       CLUSTER-IP       EXTERNAL-IP   PORT(S)        AGE
service/kube-bench-adapter   NodePort   34.118.237.228   <none>        80:32240/TCP   6m28s

NAME                               SCHEDULE   SUSPEND   ACTIVE   LAST SCHEDULE   AGE
cronjob.batch/kube-bench-adapter   @weekly    False     0        <none>          6m29s

NAME                                         COMPLETIONS   DURATION   AGE
job.batch/kube-bench-adapter-startup-c12ab   1/1           6s         6m29s
rootxrishabh@Rishabhs-MacBook-Air cfg % k get clusterpolicyreports
NAME            KIND   NAME   PASS   FAIL   WARN   ERROR   SKIP   AGE
cis-gke-1.6.0                 7      6      56     0       0      6m30s
```

**Comment by rootxrishabh** [comment]
@deboshree-b, the PR looks good overall. The tests that have been shifted to automated from manual don't have an audit command to them and could thus confuse the user. For example, in 4.1.3 **minimize wildcard use in roles and cluster role** has been shifted to automated but we don't have an audit test for it. The clusterpolicyreport shows it as a WARN with the reason "No tests defined" as shown below as well. Should we add the audit tests for it or roll it back to manual?
```
- category: CIS Google Kubernetes Engine (GKE) Benchmark
  message: Minimize wildcard use in Roles and ClusterRoles (Automated)
  policy: 4.1.3-Kubernetes Policies
  properties:
    actual_value: ""
    audit: ""
    auditConfig: ""
    auditEnv: ""
    expected_result: ""
    index: 4.1.3
    isMultiple: "false"
    reason: No tests defined
    remediation: |
      Where possible replace any use of wildcards in clusterroles and roles with specific
      objects or actions.
    test_info: |
      Where possible replace any use of wildcards in clusterroles and roles with specific
      objects or actions.
    type: automated
  result: warn
  rule: RBAC and Service Accounts
  source: kube-bench-adapter
  timestamp:
    nanos: 489880304
    seconds: 50
```
image

**Review by rootxrishabh — CHANGES_REQUESTED** [review]
Please change the folder name from gke-1.2.0 to gke-1.6.0.

**Comment by ciphx** [comment]
Adding AKS benchmarks
[CIS_Azure_Kubernetes_Service_(AKS)_Benchmark_V1.5.0_PDF.pdf](https://github.com/user-attachments/files/16862989/CIS_Azure_Kubernetes_Service_.AKS._Benchmark_V1.5.0_PDF.pdf)

**Comment by deboshree-b** [comment]
RH-1.5.0 CIS benchmarks
[CIS_Red_Hat_Openshift_Container_Platform_Benchmark_v1.5.0_PDF.pdf](https://github.com/user-attachments/files/16882949/CIS_Red_Hat_Openshift_Container_Platform_Benchmark_v1.5.0_PDF.pdf)

File: cfg/gke-1.2.0/policies.yaml [hunk 1]
```diff
@@ -20,29 +20,22 @@ groups:
...
-        text: "Minimize access to secrets (Manual)"
+        text: "Minimize access to secrets (Automated)"
         type: "manual"
         remediation: |
           Where possible, remove get, list and watch access to secret objects in the cluster.
         scored: false
 
       - id: 4.1.3
-        text: "Minimize wildcard use in Roles and ClusterRoles (Manual)"
+        text: "Minimize wildcard use in Roles and ClusterRoles (Automated)"
```
Comment (anusha94) [inline cfg/gke-1.2.0/policies.yaml]: Why changing to Automated in the text when the type is still manual?

Comment (deboshree-b) [reply cfg/gke-1.2.0/policies.yaml]: this is fixed

**Pull request closed without merging**